src/_mixins.scss

```diff
@@ -1,17 +1,42 @@
+// Inset box shadows
+@define-color highlight_color #fff;
+
+// Views and inputs
+$bg-color-1: #fff;
+
+// Background
+$bg-color-2: $SILVER_100;
+
+// Sidebars and inline toolbars
+$bg-color-3: mix($SILVER_100, $SILVER_300, $weight: 15%);
+
+// Titlebars and toolbars
+$bg-color-4: mix($SILVER_100, $SILVER_300, $weight: 30%);
+
+// Text, images, and other foreground elements
+$fg-color: $BLACK_500;
```

Comment:
I wonder if it makes more sense to do these like the shadow levels as a mixin instead of individual variables? I.e. you're setting a bg color with elevation 1, 2, 3, etc. Otherwise, I think I'd want to look at more semantic names for the variables.